website/client/components/challenges/challengeModal.vue

```diff
@@ -317,14 +317,17 @@ export default {
   methods: {
     async shown () {
       this.groups = await this.$store.dispatch('guilds:getMyGuilds');
-      await this.$store.dispatch('party:getParty');
-      const party = this.$store.state.party.data;
-      if (party._id) {
-        this.groups.push({
-          name: party.name,
-          _id: party._id,
-          privacy: 'private',
-        });
+
+      if (this.user.party) {
```

Comment:
I think this should also check this.user.party._id as user.party always exist

Reply:
it didn't exist for my test user, or it did exist but it never passed the if statement, I'll add the _id check